services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/toggles/FeatureTogglesV2Spec.groovy

```diff
@@ -112,4 +122,159 @@ class FeatureTogglesV2Spec extends HealthCheckSpecification {
         then: "Able to delete flows"
         [flow, newFlow].each { flowHelperV2.deleteFlow(it.flowId) }
     }
+
+    @Ignore("https://github.com/telstra/open-kilda/issues/2955")
+    @Tags(HARDWARE)
```

Comment:
why?

Reply:
before I tried to find a noviflow switch
for now it is fixed, `HARDWARE` tag is removed